### PR TITLE
Update redirect for APD namespace

### PR DESCRIPTION
**Pull request by dfalster**
This PR 

- simplifies the htaccess into a single file
- corrects some errors in the htaccess
- adds more info to readme

### CODE REVIEW

File: APD/README.md [hunk 1]
```diff
@@ -1,21 +1,72 @@
 This W3ID provides a persistent URI namespace for the AusTraits Plant Dictionary (APD).
+
 # AusTraits Plant Dictionary (APD)
-The AusTraits Plant Dictionary (APD) is a vocabulary that formally defines the more than 500 plant trait concepts used by [AusTraits](https://austraits.org), a database of Australian plant traits. Over time it is expected to expand to include additional trait concepts. Each trait definition has been reviewed by multiple people and includes references and links to identical/similar traits in other trait databases whenever possible. The APD includes machine-readable formats and endpoints, allowing the traits to be readily re-used by other databases.
 
-## Vocabularies
+The AusTraits Plant Dictionary (APD) is a vocabulary that formally defines the more than 500 plant trait concepts. It was developed in conjunction with [AusTraits](https://austraits.org), a database of Australian plant traits.
```
Comment (TallTed) [inline APD/README.md:5]:
```suggestion
The AusTraits Plant Dictionary (APD) is a vocabulary that formally defines more than 500 plant trait concepts. It was developed in conjunction with [AusTraits](https://austraits.org), a database of Australian plant traits. 
```

File: APD/README.md [hunk 2]
```diff
@@ -1,21 +1,72 @@
...
 ## Homepage
 
 * https://w3id.org/APD/
-* http://austraits.org
 * https://github.com/traitecoevo/APD/
+* http://austraits.org/
+
+## Design
+
+- the .htaccess code is written to process calls to variants of the machine-readable representations before any queries to html, i.e. *.ttl, *.nq, *.nt, *.json before *.html. This reduces the number of RewriteCond statements needed in the html section.
```
Comment (TallTed) [inline APD/README.md:26]:
```suggestion
- the `.htaccess` code is written to process calls to variants of the machine-readable representations before any queries to `html`, i.e., `*.ttl`, `*.nq`, `*.nt`, and `*.json`, before `*.html`. This reduces the number of `RewriteCond` statements needed in the `html` section. 
```

File: APD/README.md [hunk 3]
```diff
@@ -1,21 +1,72 @@
...
 
 * https://w3id.org/APD/
-* http://austraits.org
 * https://github.com/traitecoevo/APD/
+* http://austraits.org/
+
+## Design
+
+- the .htaccess code is written to process calls to variants of the machine-readable representations before any queries to html, i.e. *.ttl, *.nq, *.nt, *.json before *.html. This reduces the number of RewriteCond statements needed in the html section. 
+- tested with https://htaccess.madewithlove.com (using https://w3id.org/ as the base URL, using https://w3id.org/APD/ won't work as code is written as though you're already in that subfolder)
```
Comment (TallTed) [inline APD/README.md:27]:
```suggestion
- tested with https://htaccess.madewithlove.com (using `https://w3id.org/` as the base URL; `https://w3id.org/APD/` won't work, as code is written as though you're already in that subfolder)
```

File: APD/README.md [hunk 4]
```diff
@@ -1,21 +1,72 @@
...
+- the following flags are used
+  - [R=303] A redirect: when a server responds with a 303 status code, it provides a `Location` pointing to a different URI. The client, upon receiving a 303 response, automatically makes a GET request to the URI specified in the `Location` header.  
+  - [L]	If the previous conditions pass this rule is executed and no more evaluation is done. 
+  - [NE] noescape Prevent special characters being converted to their hexcode equivalent, needed for links including a `#`
```
Comment (TallTed) [inline APD/README.md:31]:
```suggestion
  - `[R=303]` — A redirect. When a server responds with a `303` status code, it provides a `Location` pointing to a different URI. The client, upon receiving a `303` response, automatically makes a `GET` request to the URI specified in the `Location` header.  
  - `[L]` — If the previous conditions pass, this rule is executed, and no more evaluation is done. 
  - `[NE]` — (noescape) Prevents conversion of special characters to their hexcode equivalent; needed for links including a `#`.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Tweaks for clarity

**Comment by dgarijo** [comment]
@dfalster please approve the proposed changes to the READMe file

**Comment by dfalster** [comment]
Thanks @dgarijo, Approved 

Thanks for the suggestions @TallTed

**Pull request merged**